package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Let's use semantic versioning please 🙏. This is a new feature, so we should do a minor version bump.
```suggestion
  "version": "1.4.0",
```